### PR TITLE
Release 1.42.0

### PR DESCRIPTION
**Pull request by mmlb**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.42.0] - 2020-02-14
+### Added
+- Capturing of hardware_reservation, spot_price_max, termination_time, and provisioning_percentage to Device
+- Support for passing custom_data when creating a device
+- Support for creating project ssh keys
+### Fixed
```
Comment (grahamc) [inline CHANGELOG.md]:
```suggestion
 - Capturing of available_in to Plan
### Fixed
```

Comment (mmlb) [reply CHANGELOG.md]: heh, you see!

**Comment by grahamc** [comment]
:tada:

**Comment by mmlb** [comment]
https://pypi.org/project/packet-python/1.42.0/

**Pull request merged**